### PR TITLE
[Import] CAES-1613: Reset state for new import

### PR DESCRIPTION
**Pull request by sa715**
Signed-off-by: Sergey Alekseyev <sa@4xxi.com>

### CODE REVIEW

File: packages/containers/Import/Import.js [hunk 1]
```diff
@@ -190,6 +193,17 @@ class Import extends Component {
...
           onClickStep={this.handleClickStep}
         />
+        {currentStep === IMPORTING_STEP && (
+          <ButtonWrapper>
+            <Button
+              color="white"
+              disabled={importProgress < IMPORT_PROGRESS_THRESHOLD}
+              onClick={this.handleCancelFlow}
+            >
+              IMPORT ONE MORE *.CSV
```
Comment (cu4xxi) [inline packages/containers/Import/Import.js:203]: Let's write text in lowerCase with the first letter - upper.
Button component has default property `text-transform: uppercase;`.
So `Import one more *.csv` will be better.

File: packages/containers/Import/Steps/ImportingStep/ImportingStep.js [hunk 2]
```diff
@@ -64,16 +58,9 @@ const ImportingStep = ({ progress, onClickToDashboard, onClickToStart }) => (
...
-      >
-        IMPORT ONE MORE *.CSV
-      </StyledButton>      
       <Button
         color="white"
-        disabled={progress < PROGRESS_THRESHOLD}
+        disabled={progress < IMPORT_PROGRESS_THRESHOLD}
         onClick={onClickToDashboard}
       >
         GO TO DASHBOARD
```
Comment (cu4xxi) [inline packages/containers/Import/Steps/ImportingStep/ImportingStep.js:66]: I know, this is not your code, but could you pls change to `Go to dashboard` too?

File: packages/containers/Import/Import.js (hunk 1 above)
Comment (sa715) [reply packages/containers/Import/Import.js]: Done

File: packages/containers/Import/Steps/ImportingStep/ImportingStep.js (hunk 2 above)
Comment (sa715) [reply packages/containers/Import/Steps/ImportingStep/ImportingStep.js]: Done

**Review by cu4xxi — APPROVED** [review]

**Review by beshkenadze — APPROVED** [review]

**Pull request merged**